src/activities/AddQueryElements.ts

```diff
@@ -344,8 +344,17 @@ export default class AddQueryElements implements IActivityHandler {
                 );
                 items = collItems.items;
             } else {
+                const filterDistinctValue = Array.from(
+                    new Set(results.features.map(
+                        (item: __esri.Graphic) => {
+                            const clone = item.clone();
+                            clone.attributes = { [searchField.field]: item.attributes[searchField.field] };
+                            return clone;
```

Comment:
How does this make it distinct? Doesn't the `clone()` ensure that different instances are passed to the `Set`, so it won't actually collapse any duplicates??